packages/core/admin/utils/plugins.js

```diff
@@ -112,7 +112,16 @@ const filterPluginsByAdminEntry = (plugin) => {
     );
 
     if (!isLocalPluginWithLegacyAdminFile) {
-      const isModuleWithFE = require.resolve(`${plugin.pathToPlugin}/strapi-admin`);
+      let packageName = plugin.info.packageName
```

Comment:
Why have you chosen this as the primary way? My concern is this won't work with plugins local to a user's strapi-application?

Reply:
I pushed a new variation that only fixes the paths on windows.